Mage/src/main/java/mage/abilities/effects/common/DraftFromSpellbookThenExilePlayThisTurnEffect.java

```diff
@@ -0,0 +1,82 @@
+package mage.abilities.effects.common;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.asthought.PlayFromNotOwnHandZoneTargetEffect;
+import mage.cards.Card;
+import mage.cards.repository.CardCriteria;
+import mage.cards.repository.CardInfo;
+import mage.cards.repository.CardRepository;
+import mage.choices.Choice;
+import mage.choices.ChoiceHintType;
+import mage.choices.ChoiceImpl;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.TargetController;
+import mage.game.Game;
+import mage.players.Player;
+import mage.util.RandomUtil;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * @author Sidorovich77
+ */
+public class DraftFromSpellbookThenExilePlayThisTurnEffect extends OneShotEffect {
```

Comment:
@xenohedron 
As I remember play and cast now are same due oracle changes few releases ago? If true then no needs in duplicated class

Reply:
I'm pretty sure there's still a difference - but shouldn't need a duplicated class anyway.